safety_gymnasium/wrappers/gymnasium_conversion.py

```diff
@@ -23,6 +23,9 @@ class SafetyGymnasium2Gymnasium(gymnasium.Wrapper):
 
     def step(self, action: ActType):
         obs, reward, cost, terminated, truncated, info = super().step(action)
+        assert (
+            'cost' not in info
+        ), 'The info dict already contains a cost. Overwriting it may cause unexpected behavior.'
```

Comment:
Maybe use `np.allclose` rather than forbid the key.

Reply:
make sense